src/core/lib/gprpp/status_helper.cc

```diff
@@ -200,6 +214,29 @@ absl::optional<std::string> StatusGetStr(const absl::Status& status,
   return {};
 }
 
+void StatusSetTime(absl::Status* status, StatusTimeProperty key,
+                   absl::Time time) {
+  status->SetPayload(GetStatusTimePropertyUrl(key),
+                     absl::Cord(absl::string_view(
+                         reinterpret_cast<const char*>(&time), sizeof(time))));
```

Comment:
Instead of jumping through hoops to store the contents of an `absl::Time` object as a string, would it be more efficient to just store times as `grpc_millis`?

Reply:
I don't think that they have different performance since absl::Time is 16 bytes bigs and grpc_millis is 8 bytes big on 64bits. Moreover grpc_millis isn't in gpr.